src/app/types/any-for-moda-data.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line
+export type AnyForModalData = any;
```

Comment:
I'm not sure that variable name like AnyForModalData is quiet common and understandable.
Please rename it to "Any", "CustomAny", "AnyType", smth like that.
Also please review filename.